back/tests/cmd/openbsrv-integration/integration_test.go

```diff
@@ -128,7 +199,8 @@ func userSvcAddAndFndUsersFn(ctx context.Context, conn *grpc.ClientConn, clnt pb
 		}
 
 		for _, tc := range tests {
-			r, err := clnt.AddUser(ctx, tc.addReq)
+			t.Log(tc.desc)
```

Comment:
I suggest dropping noise from tests, but, if you insist, I'm happy to accept it.